qabelbox/src/androidTest/java/de/qabel/qabelbox/box/provider/MockBoxProvider.kt

```diff
@@ -6,15 +6,15 @@ import android.content.pm.ProviderInfo
 import de.qabel.qabelbox.BuildConfig
 import de.qabel.qabelbox.TestConstants
 import de.qabel.qabelbox.box.dto.BoxPath
-import de.qabel.qabelbox.box.interactor.ProviderUseCase
+import de.qabel.qabelbox.box.interactor.Provider
 
 class MockBoxProvider : BoxProvider() {
 
 
     override fun inject() {
     }
 
-    fun injectProvider(provider: ProviderUseCase) {
+    fun injectProvider(provider: Provider) {
```

Comment:
I don't want to play with words here but in general: `provider` might be a good name for a variable/parameter but it's not that a good class name imho. It's like naming something `Factory` ... you know it's a factory but what kind? What does it produce? And what does this provider provide?